src/StreamJsonRpc.Shared/DataContracts/JsonRpcMessage.cs

```diff
@@ -120,6 +127,13 @@ public object[] GetParameters(ParameterInfo[] parameterInfos, JsonSerializer jso
                 return this.Parameters.ToObject<object[]>();
             }
 
+            if (parameterInfos.Length == 1 && parameterInfos[0].ParameterType == typeof(JToken))
+            {
+                // We want to ignore methods that only have one parameter and the parameter type is JToken.
+                // This is reserved to be called when the parameter is passed as an object.
+                return null;
```

Comment:
So if I leave this out I get an exception in the following case: Server method have 2 methods that both take only one argument.  One method takes JToken, the other takes string.  What ends up happening is both of them are matched when InvokeAsync is called and the code ends up throwing an exception due to multiple methods found.  Is this an existing bug then?